app/src/test/java/com/github/mobile/ui/user/NewsListAdapterTest.java

```diff
@@ -0,0 +1,13 @@
+package com.github.mobile.ui.user;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class NewsListAdapterTest {
+
+    @Test
+    public void testUpdate() throws Exception {
+        assertEquals(1, 1);
```

Comment:
Removed this test and added other 18 unit tests to test the icon and
formatted text for each event. .

On Sat, Mar 21, 2015 at 1:54 AM, Fadil Sutomo <notifications@github.com>
wrote:

> In app/src/test/java/com/github/mobile/ui/user/NewsListAdapterTest.java
> <https://github.com/github/android/pull/743#discussion_r26889460>:
>
> > @@ -0,0 +1,13 @@
> > +package com.github.mobile.ui.user;
> > +
> > +import org.junit.Test;
> > +
> > +import static org.junit.Assert.assertEquals;
> > +
> > +public class NewsListAdapterTest {
> > +
> > +    @Test
> > +    public void testUpdate() throws Exception {
> > +        assertEquals(1, 1);
>
> What exactly are we testing here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/github/android/pull/743/files#r26889460>.
>